src/main/java/com/laytonsmith/core/functions/StringHandling.java

```diff
@@ -1157,7 +1158,7 @@ public Construct exec(Target t, Environment environment, Construct... args) thro
 				params[i] = convertArgument(arg, c, i, t);
 			}
 			//Ok, done.
-			return new CString(String.format(formatString, params), t);
+			return new CString(String.format(Locale.US, formatString, params), t);
```

Comment:
This fixes the failing unit test on all OS'es that use 1,23 as decimal seperator instead of 1.23.